manifest.json

```diff
@@ -1,20 +1,46 @@
 {
-  "name": "My App",
-  "short_name": "My App",
+  "name": "Request Yo Racks",
+  "short_name": "RYR",
   "start_url": "./?homescreen=1",
   "display": "standalone",
   "theme_color": "#3f51b5",
   "background_color": "#3f51b5",
   "icons": [
-    {
-      "src": "images/manifest/icon-192x192.png",
-      "sizes": "192x192",
-      "type": "image/png"
-    },
-    {
-      "src": "images/manifest/icon-512x512.png",
-      "sizes": "512x512",
-      "type": "image/png"
-    }
-  ]
+  {
+   "src": "images/manifest/android-icon-36x36.png",
```

Comment:
There is no need to duplicate the icons. You can directly use the ones we have in the `images/logos` folder.

Reply:
Okay. I'll go ahead and change that now